Suggestion for a shorter abstract (#40)
intarchboard/draft-path-signals-collaboration: draft-iab-path-signals-collaboration.md

   
--- abstract

This document discusses principles for designing mechanisms that use or provide
path signals, and calls for standards action in specific valuable cases.
RFC 8558 describes path signals as messages to or from on-path elements,
and points out that visible information will be used whether it is

> intended as a signal or not. The principles in this document apply to
> explicit path signals, which are encouraged to be authenticated and include

Author: ```suggestion
intended as a signal or not. The principles in this document are intended as
guidance for the design of explicit path signals, which are encouraged to be 
authenticated and include
```

> a minimal set of parties and minimal information. These principles can

Reviewer: ```suggestion
a minimal set of parties and minimize information sharing. These principles can
```

> a minimal set of parties and minimal information. These principles can
> be achieved through encryption of information and establishing trust relationships
> between entities on a path.

Author: ```suggestion
a minimal set of parties and minimal information. These principles can
be achieved through mechanisms like encryption of information and
establishing trust relationships between entities on a path.
```
